features/storage/csi_clone.feature

```diff
@@ -0,0 +1,67 @@
+Feature: CSI clone testing related feature
+  """
+  This test can only work on a Cluster which has CSI driver deployed, OSP has csi deploy by default from 4.7. 
+  When deploy a cluster which has no csi deployed, please use enable_tpnu_featureset: "yes" in LAUNCHER_VARS.
+  From 4.7, OSP CSI driver was installed by default. Its storage class name is standard-csi.
+  Use below command can temporarily add cs class(oc get sc, can find all deployed stroage class):
+  oc patch featuregate cluster -p '{"spec": {"featureSet": "TechPreviewNoUpgrade"}}' --type merge
+  """
```

Comment:
Thanks for this comment, I like to see this in our tests, this is a good practice.

Reply:
@JianLi-RH That's good to add such info, as we supposed that csi driver is installed already, we could only mention the platform, OCP verison and storageclass name, I think no need to mention the feature gate and "enable_tpnu_featureset" (a flag of flexy-installation template)

Reply:
@duanwei33 OK, i will remove "When deploy a cluster which has no csi deployed, please use enable_tpnu_featureset: "yes" in LAUNCHER_VARS." from comments.

Reply:
@JianLi-RH Please also remove the line6~7 for the cmd. It will not be covered in our test case.

Reply:
@duanwei33 Line 6-7 just a reminder for whose environment does not have storage class installed, so I think keep it here is proper, do you think so?

Reply:
@JianLi-RH Hmm, as I said, we could add the information we supported(like platform, OCP verison and storageclass name), means we will not cover the untouched cases,  and will not perform such command in our case.

Reply:
@duanwei33 Yes, its my fault, i did not noticed that. I will remove these lines.